### PR TITLE
Docker improvements by malteA

### PR DESCRIPTION
**Pull request by waznico**
Thanks to Malte there're some improvements within the docker process.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,10 +30,13 @@
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
     "rxjs": "^6.5.4",
-    "swagger-ui-express": "^4.1.3"
```
Comment (waznico) [inline package.json]: @malteA Why was it removed? Can you please reinsert the dependencies for swagger, nestjs/cli etc?

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿version: '3'
+services:
+  app:
+    build: .
+    ports:
+      - '5000:3000'
```
Comment (waznico) [inline docker-compose.yaml]: Why was the port changed from 3000 to 5000? Is it necessary? If not I'd prefer to continue with the old port to ensure compatiblity.

File: package.json (hunk 1 above)
Comment (malteA) [reply package.json]: I just moved them as you can see, nestjs/cli is needed in "dependencies" not in "devDependencies"

Comment (waznico) [reply package.json]: Yeah thanks for the quick response! I'll approve the PR :wink:

**Pull request merged**